test/cram/test.t

```diff
@@ -379,7 +379,15 @@ This is a cram test for the new executable.
   
   Type 'h' to go to the help panel, 'q' to quit, 'n' to go to the next operation, 'p' to go to the previous operation. Press 't' to toggle view mode.
 
+# Test unified view with text markers (colorless)
+$ dummy_terminal example.diff r
```

Comment:
The lines starting in `$` must be indented by two space if you want them to be executed.